microprofile/lra/coordinator/src/main/java/org/wildfly/extension/microprofile/lra/coordinator/service/LRARecoveryService.java

```diff
@@ -59,6 +77,15 @@ public synchronized void start(final StartContext context) throws StartException
         // installing LRA recovery types
         Implementations.install();
 
+        // Add ObjStore type handlers
+        objStoreBrowser = ObjStoreBrowser.getInstance();
+
+        for (String[] typeAndBean : LRA_OSB_TYPES) {
+            String typeName = typeAndBean[0].replaceAll("/", File.separator);
```

Comment:
This fails on Windows. Probably because File.separator is `\'. See the String.replaceAll javadoc for more details.

Reply:
Thanks @bstansberry , I see. I replaced replaceAll with replace so the regex  is not used.